CODING_STANDARDS.md

```diff
@@ -167,6 +167,14 @@ be part of the user-facing API. This typically applies to symbols used in templa
 Additionally, the `@docs-private` JsDoc annotation can be used to hide any symbol from the public
 API docs.
 
+
+#### Getters and Setters
+* Always use a `readonly` property instead of a getter (with no setter) when possible.
+* Avoid long getters and setters. If the logic of an accessor would take more than three lines,
+introduce a new method to contain the logic.
+* A getter should also immediately precede its corresponding setter.
```

Comment:
If `@Input` is on setter, it should come first right? This is saying that getters precede the setter (getters come first)

Reply:
Getters come first, `@input` always goes on the getter.